brayns/common/scene/Scene.cpp

```diff
@@ -111,6 +111,9 @@ void Scene::clearLights()
 
 size_t Scene::addModel(ModelDescriptorPtr model)
 {
+    if (model->getModel().empty())
```

Comment:
The same applies for submodels, so please add this to Model::addModel() as well. I'm about to unify this somewhat duplication of code to add models, but for the time being it would be good to have those checks.